### PR TITLE
feat: move to skeleton v2

### PR DESCRIPTION
**Pull request by baseplate-admin**

### CODE REVIEW

File: src/routes/+layout.svelte [hunk 1]
```diff
@@ -18,16 +18,17 @@
     import { navbar_middle_section_variant } from "$store/navbar";
     import { theme } from "$store/theme";
     // Most of your app wide CSS should be put in this file
-    import "../app.css";
+    import "../app.postcss";
```
Comment (tokitou-san) [inline src/routes/+layout.svelte:21]: Screenshot from 2023-08-16 05-31-10

Unfortunately custom fonts are not working.

Comment (AdrianGonz97) [reply src/routes/+layout.svelte:21]: Hi @tokitou-san! This is an issue with `@skeletonlabs/tw-plugin` version `0.0.1-rc.0`. Updating to `0.0.1-rc.3` should fix it!

Comment (tokitou-san) [reply src/routes/+layout.svelte:21]: > Hi @tokitou-san! This is an issue with @skeletonlabs/tw-plugin version 0.0.1-rc.0. Updating to 0.0.1-rc.3 should fix 

Hi @AdrianGonz97, issue fixed.
Thanks a lot :)

**Comment by baseplate-admin** [comment]
@tokitou-san, Does everything look good to you?

If that's the case we can merge

**Pull request merged**